docs/source/concept_models/vivarium_iv_iron/concept_model.rst

```diff
@@ -354,21 +349,34 @@ Locations of interest to this project:
 
 - Sub-Saharan Africa (location_type=superregion; location_id=166)
 - South Asia (location_type=region; location_id=159)
-- All low and middle income countries
+- All low and middle income countries:
 
-.. todo::
+  - World bank lower middle income (location_type=region; location_id=44577)
+  - World bank low income (location_type=region; location_id=44578)
 
-  Compile all national location IDs included in each of these regional locations of interest using the GBD shared function :code:`get_location_metadata`, `documented here <https://scicomp-docs.ihme.washington.edu/db_queries/current/get_location_metadata.html>`_.
-  
-.. todo::
+National-level locations included in each of these locations of interest `can be found here <https://github.com/ihmeuw/vivarium_research_iv_iron/tree/main/locations>`_.
+
+Location aggregation
+^^^^^^^^^^^^^^^^^^^^^^
+
+For GBD outcomes that do not have regional-level estimates (e.g. covariates), the following strategy should be followed:
+
+#. Pull estimates specific to each national-level location_id included in the region of interest (can be found in csv files linked above)
+#. Pull population estimates for each national-level location_id included in the region of interest
+#. At the draw-level, caclulate a population-weighted average estimate across all national locations within the region of interest, like so:
+
+.. math::
 
-    Determine wich location ID and hierarchy to use for LMICs (commonwealth versus world bank classifications... need to determine which have GBD outcomes of interest) 
+  estimate_\text{regional} = \frac{\sum_{n=1}^{n} population_\text{national} * estimate_\text{national}}{\sum_{n=1}^{n} population_\text{national}}
 
 .. _iviron4.2.1:
 
 4.2.1 Population description
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+* **Population size:** 100,000
+* **Cohort type:** open
```

Comment:
Probably easier to start with women three years below reproductive age and keep the cohort closed for this version